index.html

```diff
@@ -1,55 +1,105 @@
 <!DOCTYPE html>
-<html lang="en">
```

Comment:
I see you rewrite your previous homework with the new one. So you have the same content in both index.html and hw7.html.

Try to get the index.html content back